README.md

```diff
@@ -1,70 +1,172 @@
+In order to run this project you need:
 
-### Analyzing the Bundle Size
+- Basic knowledge of react, redux and CSS
+- Basic understanding of Linters
+- Basic understanding of Github
+- Your machine should have Chrome and VSCode installed
 
-This section has moved here: [https://facebook.github.io/create-react-app/docs/analyzing-the-bundle-size](https://facebook.github.io/create-react-app/docs/analyzing-the-bundle-size)
+### Setup
 
-### Making a Progressive Web App
+Clone this repository to your desired folder:
 
-This section has moved here: [https://facebook.github.io/create-react-app/docs/making-a-progressive-web-app](https://facebook.github.io/create-react-app/docs/making-a-progressive-web-app)
+- git clone https://github.com/aradradev/bookstore.git
 
-### Advanced Configuration
+### Install
 
-This section has moved here: [https://facebook.github.io/create-react-app/docs/advanced-configuration](https://facebook.github.io/create-react-app/docs/advanced-configuration)
+Install this project with:
+
+- You need to install Chrome if you want to see it working locally
+
+### Usage
+
+To run the project, execute the following command:
+
+- open index.html with Chrome or other browsers.
+
+### Run tests
+
+To run tests, run the following command:
+
+- npm install
+- npm start
+- npx stylelint "\*_/_.{css,scss}"
+- npx hint .
+- npx eslint . --fix
 
 ### Deployment
 
-This section has moved here: [https://facebook.github.io/create-react-app/docs/deployment](https://facebook.github.io/create-react-app/docs/deployment)
+You can deploy any service you want.
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- AUTHORS -->
+
+## 👥 Authors <a name="authors"></a>
+
+👤 **Jalloh**
+
+- GitHub: [aradradev](https://github.com/aradradev)
+- Twitter: [@abdoul_2023](https://twitter.com/Abdoul_2023)
+- LinkedIn: [Abdourahmane Diallo](https://www.linkedin.com/in/abdoul-ramane-diallo-15b2a2262/)
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- FUTURE FEATURES -->
+
+## 🔭 Future Features <a name="future-features"></a>
+
+- [ ] **Styling**
+- [ ] **Animation**
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- CONTRIBUTING -->
+
+## 🤝 Contributing <a name="contributing"></a>
+
+Contributions, issues, and feature requests are welcome!
+
+Feel free to check the [issues page](https://github.com/aradradev/add-awesome-Book/issues).
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- SUPPORT -->
+
+## ⭐️ Show your support <a name="support"></a>
+
+If you like this project give it a ⭐️
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- ACKNOWLEDGEMENTS -->
+
+## 🙏 Acknowledgments <a name="acknowledgements"></a>
+
+I would like to say thank all co-authors and contributors
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- LICENSE -->
+
+## 📝 License <a name="license"></a>
```

Comment:
Nice job so far, everything looks great👍. Just a little issue with your `mit` license link which is currently broken😉. Kindly do well to fix this as this can lead to a frustrating user experience and give your project a less professional look👍.

<img width="776" alt="image" src="https://github.com/aradradev/bookstore/assets/64643995/5c562859-d512-4141-962f-34f3d6bd418d">